mydoc/mydoc_cmor3_api.md

```diff
@@ -333,6 +333,63 @@ Returns upon success:
 {:.coral} 
 
 
+---
+
+### cmor_set_crs()
+
+Fortran: error_flag = cmor_set_crs(grid_id, mapping_name, parameter_names, parameter_values, parameter_units, text_parameter_names, text_parameter_values)
+{:.green}
+
+C: error_flag = cmor_set_crs(int grid_id, char *mapping_name, int nparameters, char **parameter_names, int lparameters, double parameter_values[], char **parameter_units, int lunits, int ntextparameters, char **text_parameter_names, int ltextparameters, char *text_parameter_values, int text_parameter_length[]) 
+{:.blue} 
+
+Python: set_crs(grid_id, mapping_name, parameter_names, parameter_values=None, parameter_units=None) 
+{:.coral} 
+
+
+*Description*: Define the Coordinate Reference System (CRS) variable that contains grid mapping parameters associated with a grid (see CF conventions for more info on which parameters to set). Similar to [cmor_set_grid_mapping](#cmor_set_grid_mapping), but allows for the setting of text-based parameters.
+
+*Note on `crs_wkt`*: This function allows for the setting of a [`crs_wkt`](https://cfconventions.org/cf-conventions/cf-conventions.html#use-of-the-crs-well-known-text-format){:target="_blank"} text attribute, meant to hold a Well-known Text (WKT) formatted string. However, CMOR will not validate the contents of this string.
+
+
+*Arguments*:
+
+  * **grid_id** = the “handle” returned by a previous call to cmor_grid, indicating which grid the mapping parameters should be associated with.
+
+  * **mapping_name** = name of the mapping (see CF conventions). This name dictates which parameters should be set and for some parameters restricts their possible values or range. New mapping names can be added via MIP tables.
```

Comment:
It would be useful to add ths direct link to the CF grid mapping names that we accept - http://cfconventions.org/Data/cf-conventions/cf-conventions-1.12/cf-conventions.html#appendix-grid-mappings is a good link, not sure we have a shorter list-like version of these names, we do in code, it would be useful to be explicit within these docs, so that I user doesn't go an attempt to define a grid mapping that the code will not enable

Reply:
Sorry @mauzey1 I just spotted this comment. I do think it would be helpful to enumerate each of the valid/accepted mapping names, if we have this in code, then listing these would be useful - we already have the link to the CF grid mapping names, but pulling these out of code to be explicit in docs is useful